content/en/blog/recap-2022/index.md

```diff
@@ -0,0 +1,136 @@
+- [David Windsor](https://github.com/dwindsor), Secureworks
+- [Federico Di Pierro](https://github.com/fededp), Sysdig
+- [Frank Jogeleit](https://github.com/fjogeleit), LOVOO
+- [Fred Araujo](https://github.com/araujof), IBM
+- [Grzegorz Nosek](https://github.com/gnosek), Sysdig
+- [Hendrik Brueckner](https://github.com/hbrueckner), IBM
+- [Jason Dellaluce](https://github.com/jasondellaluce), Sysdig
+- [Jonah Jones](https://github.com/jonahjon), Amazon
+- [Leonardo Di Donato](https://github.com/leodido), Independent
+- [Leonardo Grasso](https://github.com/leogr), Sysdig
+- [Logan Bond](https://github.com/exoner4ted), Secureworks
+- [Loris Degioanni](https://github.com/ldegio), Sysdig
+- [Luca Guerra](https://github.com/lucaguerra), Sysdig
+- [Mark Stemm](https://github.com/mstemm), Sysdig
+- [Massimiliano Giovagnoli](https://github.com/maxgio92), Clastix
+- [Mauro Ezequiel Moltrasio](https://github.com/molter73), RedHat
+- [Michele Zuccala](https://github.com/zuc), Sysdig
+- [Radu Andries](https://github.com/admiral0), Independent
+- [Teryl Taylor](https://github.com/terylt), IBM
+- [Thomas Labarussias](https://github.com/issif), Sysdig
+
+### New and noteworthy in Falco land 
+The Falco community was busy in 2022, figuring out how to better serve contributors, developing new features, fixing bugs, and taking the project to new heights. Here is a recap of key milestones reached this year: 
+
+- Falco’s governance underwent a facelift this year. Why is this important? As a CNCF project, Falco is committed to promoting a healthy community of contributors and maintainers from multiple vendors. In aid of this, the core maintainers formerly announced & published these updates to help contributors better serve the community. To read the full announcement, see [updating Falco Governance](https://falco.org/blog/updating-falco-governance/).
+At the beginning of 2022, Falco introduced a game changing feature, Falco Plugins. They allow Falco to monitor and trigger alerts for any kind of event. Since the launch of the new plugin framework the Falco community has collaborated to create plugins for GitHub, AWS CloudTrail and Okta.
+
+A plugin also replaced the way Falco consumes the Audit Logs generated by a K8s API server. With these plugins, Falco covers more in depth the aspects of your infrastructure and allows you to use a single syntax for rules. Our adopters asked us for a way to monitor K8s Audit Logs and the previous implementation used an internal web server to receive the logs from the Kubernetes API. This method didn't support clusters managed by cloud providers, such as EKS, AKS, or GKE as they had to capture the Audit Logs for their own usage and then add them to their log aggregators. This situation is now solved thanks to the plugin framework and we're proud to have announced the first release of the plugin for EKS Audit Logs!!! Read the full blog [here!](https://falco.org/blog/k8saudit-eks-plugin/)
```

Comment:
the format is messed up and needs to be indented
this should also be part of the bullet from line 47

Reply:
```suggestion

```